src/components/App.js

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../styles/App.scss';
 import '../styles/Button.scss';
 import '../styles/ButtonPanel.scss';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
+import calculate from '../logic/calculate';
 
 function App() {
```

Comment:
Now you're in the React club!! Great job using hooks here instead of that "ugly" class :D :D